### DISCUSSION TITLE
Is there a way to make the game to display the inverted question mark?

### DISCUSSION THREAD

**Discussion by MaybeDavey** [Kaplas80/TF3#48] [Q&A] [answered]
Hi, I've been using your tool for quite a time ago. I'm working on a Kiwami 2 Spanish Translation and haven't had any important issues so far, except for the inability of the game to display "¿", instead, the game replaces it with a "/".

Does the app have a feature to fix this issue, or I just have to try to use a custom font?

**Comment by Kaplas80** [comment, marked as answer]
Hi! The game font doesn't have the `¿` symbol, so you have to edit the font image to add it.

**Reply by MaybeDavey** [reply]
Hey! I tried to edit the font, but the character won't show anyway. The way I did it was selecting the (?) symbol in PS, made it a Layer, and inverted it, and then I replaced an unused character in the .dds file.

Should it have worked if I typed the character (¿) instead?

![fonttt](https://user-images.githubusercontent.com/85464600/211204054-7c62a9ba-be62-45d0-a2dc-a68468388757.png)


![fontissue](https://user-images.githubusercontent.com/85464600/211204064-b1713b16-e009-4f95-8558-747d0250e2da.png)

**Reply by MaybeDavey** [reply]
Ok I just fixed the issue, it was actually very simple.

Thank you tho!
